pkg/handler/enqueue.go

```diff
@@ -33,6 +33,14 @@ var _ EventHandler = &EnqueueRequestForObject{}
 // Controllers that have associated Resources (e.g. CRDs) to reconcile the associated Resource.
 type EnqueueRequestForObject struct{}
 
+const (
+	// NamespacedNameAnnotation - annotation that will be used to get the primary resource namespaced name.
+	NamespacedNameAnnotation = "sigs.k8s.io/primary-resource"
+
+	// TypeAnnotation - annotation that will be used to verify that the primary resource is the primary resource to use.
+	TypeAnnotation = "sigs.k8s.io/primary-resource-type"
+)
+
```

Comment:
Move these to `enqueue_annotation.go` since that's the only place they're referenced.